websites/A/AnimeTW/dist/metadata.json

```diff
@@ -0,0 +1,19 @@
+{
+  "author": {
+    "name": "Giingu",
+    "id": "156481332652802048"
+  },
+  "service": "AnimeTW",
+  "description": {
+    "en": "AnimeTW is a streaming website where you can watch anime with Spanish subtitles.",
+    "zh_CN": "AnimeTW是一个视频网站，您可以在其中观看带有西班牙语字幕的动漫。"
+  },
+  "url": "animetw.net",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/NHoHFWf.jpg",
```

Comment:
Hello, can you change the logo with one that has better quality or prove that the logo is supposed to be like this resolution?

Reply:
they do not have any logo with better quality, they didnt even have a background and i had to add it

Reply:
https://animetw.net/ check the site yourself and try finding their logo on google images, theres nothing else